testcases/client-server/dnf-FTP/server

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+mkdir -p /run/fedora-networking
+/usr/bin/cp -u dummy.spec /run/fedora-networking/
+cd /run/fedora-networking
+
+rpmbuild -ba dummy.spec
+
+mkdir -p /var/ftp/pub/fedora-networking
+
+/usr/bin/cp -u \
+    /root/rpmbuild/RPMS/noarch/dummy-rpm-1.0-1.noarch.rpm \
+    /var/ftp/pub/fedora-networking
```

Comment:
Same here. It'd be good to use /run/ for all testing related stuff (overriden vsftpd config?)

Reply:
Moved.